### PR TITLE
Video update

### PR DESCRIPTION
**Pull request by apunko**

### CODE REVIEW

File: server/src/lib/donor/donor-api.ts [hunk 1]
```diff
@@ -105,7 +105,7 @@ export default class DonorAPI {
     router.get('/:districtId/media', this.getDistrictMedia);
     router.post(
       '/media',
-      bodyParser.raw({ type: 'image/*', limit: '12288kb' }),
+      bodyParser.raw({ type: '*/*', limit: '24576kb' }),
```
Comment (b83) [inline server/src/lib/donor/donor-api.ts:108]: please increase to 200mb

File: server/src/lib/model/db/migrations/20221003180237-donar-by-destrict-video-subtype.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export const up = async function (db: any): Promise<any> {
+  return db.runSql(
+    `
+      ALTER TABLE media MODIFY COLUMN subtype ENUM('mova-mark', 'literature-mark', 'dictation-mark', 'video') AFTER type;
```
Comment (b83) [inline server/src/lib/model/db/migrations/20221003180237-donar-by-destrict-video-subtype.ts:4]: I'd sort them alphabetically :)

**Review by b83 — COMMENTED** [review]
Усё кул. толькі раз ты ўжо тут, то было б добра павялічыць ліміт на 200мб відэа

File: server/src/lib/model/db/migrations/20221003180237-donar-by-destrict-video-subtype.ts (hunk 2 above)
Comment (apunko) [reply server/src/lib/model/db/migrations/20221003180237-donar-by-destrict-video-subtype.ts:4]: it's impossible to change order after we have rows in our table
strings will transform in numbers under the hood and stored in db

Comment (b83) [reply server/src/lib/model/db/migrations/20221003180237-donar-by-destrict-video-subtype.ts:4]: Actually I've done that for a type column and it worked :)

**Pull request merged**